app/core/rate_limiter.py

```
@@ -0,0 +1,168 @@
+        # Get or create token bucket for identifier
+        if identifier not in self.buckets:
+            self.buckets[identifier] = TokenBucket(
+                capacity=self.requests_per_minute,
+                refill_rate=self.requests_per_minute / 60.0  # per second
+            )
+
+        bucket = self.buckets[identifier]
+        allowed = bucket.consume()
+
+        # Calculate remaining requests and reset time
+        requests_made = self.requests_per_minute - int(bucket.tokens)
```

Ruff (0.13.1)

Local variable `requests_made` is assigned to but never used

Remove assignment to unused variable `requests_made`

(F841)